packages/website/pages/index.js

```diff
@@ -417,6 +417,19 @@ function FAQ() {
           </a>{' '}
           for details.
         </p>
+        <h2 className="chicagoflf white">
+          <HashLink id="how-long-will-data-be-stored-on-nftstorage">
+            How can I learn more about NFT best practices?
```

Comment:
Where? Maybe a british/american thing? IDK

```suggestion
            Where can I learn more about NFT best practices?
```

Reply:
Not sure if that's a regional thing? But both work well. Thanks!